### PR TITLE
Add English translation & fix typos in German strings file

### PR DESCRIPTION
**Pull request by marcauberer**
Hi,
I have created an English translation for the project.

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -193,223 +193,223 @@
...
-    <string name="submission_positive_other_warning_button">Weiter</string>
-    <string name="submission_positive_dialog_confirmation_title">Zufalls-IDs mit der App "Corona-Warn" teilen?</string>
+    <string name="submission_positive_other_warning_button">Next</string>
+    <string name="submission_positive_dialog_confirmation_title">Share random IDs with the "Corona-Warn" app?</string>
     <string name="submission_positive_dialog_confirmation_body">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</string>
-    <string name="submission_positive_dialog_confirmation_positive">Erlauben</string>
-    <string name="submission_positive_dialog_confirmation_negative">Nicht erlauben</string>
+    <string name="submission_positive_dialog_confirmation_positive">Allow</string>
+    <string name="submission_positive_dialog_confirmation_negative">Not allow</string>
```
Comment (kedar1607) [inline Corona-Warn-App/src/main/res/values/strings.xml:405]: Can this be just "Cancel" or "Don't allow"?

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -193,223 +193,223 @@
...
     ###################################### -->
     <!-- Permission Rationale Dialog -->
-    <string name="submission_qr_code_scan_permission_rationale_dialog_headline">Kamera Berechtigung benötigt</string>
+    <string name="submission_qr_code_scan_permission_rationale_dialog_headline">Camera permission required</string>
     <string name="submission_qr_code_scan_permission_rationale_dialog_body">@string/lorem_ipsum</string>
 
-    <string name="submission_qr_code_scan_permission_rationale_dialog_button_positive">Erlauben</string>
-    <string name="submission_qr_code_scan_permission_rationale_dialog_button_negative">Nicht erlauben</string>
+    <string name="submission_qr_code_scan_permission_rationale_dialog_button_positive">Allow</string>
+    <string name="submission_qr_code_scan_permission_rationale_dialog_button_negative">Not allow</string>
```
Comment (kedar1607) [inline Corona-Warn-App/src/main/res/values/strings.xml:321]: Can this be just "Cancel" or "Don't allow"?

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -193,223 +193,223 @@
...
     ###################################### -->
     <string name="onboarding_title">Onboarding</string>
     <string name="onboarding_image_description">IS MISSING</string>
-    <string name="onboarding_button_finish">Zustimmen</string>
-    <string name="onboarding_button_disable">Nicht aktivieren</string>
-    <string name="onboarding_button_next">Weiter</string>
-    <string name="onboarding_button_start">Los Geht\'s</string>
-    <string name="onboarding_button_back_description">Zurück</string>
+    <string name="onboarding_button_finish">Agree</string>
+    <string name="onboarding_button_disable">Not enable</string>
```
Comment (kedar1607) [inline Corona-Warn-App/src/main/res/values/strings.xml:220]: May be "Disagree"?

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -193,223 +193,223 @@
...
-    <string name="risk_details_behavior_body_wash_hands">Waschen Sie Ihre Hände</string>
-    <string name="risk_details_behavior_body_wear_mask">Tragen Sie eine Schutzmaske</string>
-    <string name="risk_details_behavior_body_stay_away">Halten Sie 2 Meter Abstand zu anderen Personen</string>
-    <string name="risk_details_behavior_body_cough_sneeze">Husten und Niesen Sie in Ihre Armbeuge</string>
-    <string name="risk_details_behavior_increased_body_1">- Ihren Hausartzt/Ihre Hausärtztin</string>
-    <string name="risk_details_behavior_increased_body_2">- Den Kassenärtzlichen Notdienst (Tel: 116117)</string>
-    <string name="risk_details_behavior_increased_body_3">- Ihr Gesundheitsamt</string>
-    <string name="risk_details_headline_infection_risk">Infektionsrisiko</string>
-    <string name="risk_details_body_infection_risk">So wird Ihr Risiko ermittelt</string>
+    <string name="risk_details_headline_behavior">Behave correctly</string>
```
Comment (kedar1607) [inline Corona-Warn-App/src/main/res/values/strings.xml:196]: I'm not sure what it means. But "Take appropriate actions" seems more appropriate here.

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -113,77 +113,77 @@
...
     <!-- ####################################
                    Main - Share
     ###################################### -->
-    <string name="main_share_title">Corona-Warn-App teilen</string>
-    <string name="main_share_headline">Gemeinsam Corona bekämpfen</string>
-    <string name="main_share_body">Je mehr Menschen mitmachen, desto schneller durchbrechen wir dir Infektionsketten. Laden Sie Familie, Freunde und Bekannte zum Download ein!</string>
-    <string name="main_share_button">Download-Link versenden</string>
+    <string name="main_share_title">Share the Corona-Warn-App</string>
+    <string name="main_share_headline">Fight against Corona together</string>
+    <string name="main_share_body">The more people who participate, the faster we break the chains of infection. Invite family, friends and acquaintances to download!</string>
```
Comment (kedar1607) [inline Corona-Warn-App/src/main/res/values/strings.xml:182]: "who" is unnecessary here.

**Review by kedar1607 — COMMENTED** [review]
Great work! Made some comments. Please let me know if worth making those changes. Thanks.

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 2 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values/strings.xml]: Fixed. Changed it to "Don't allow"

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 5 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values/strings.xml]: Fixed. Removed the "who"

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 3 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values/strings.xml:220]: I would translate it word by word. In the German version the string is "Nciht aktivieren", which definitely not means "Disagree"

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 1 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values/strings.xml]: Fixed. Changed it to "Don't allow"

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 4 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values/strings.xml:196]: Same here. I would leave it like this

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 3 above)
Comment (kedar1607) [reply Corona-Warn-App/src/main/res/values/strings.xml:220]: Then can we change this also to "Don't enable?"

**Review by kedar1607 — DISMISSED** [review]
LGTM

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 3 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values/strings.xml:220]: Yes. I'll just change it

**Review by kedar1607 — DISMISSED** [review]

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -113,77 +113,77 @@
...
-    <string name="submission_status_card_body_negative">Sie wurden negativ auf SARS-CoV-2 getestet.</string>
-    <string name="submission_status_card_button_unregistered">Informieren &amp; mitmachen</string>
-    <string name="submission_status_card_button_show_results">Ergebnis anzeigen</string>
-    <string name="submission_status_card_button_show_details">Details anzeigen</string>
+    <string name="submission_status_card_title_available">Your result is not yet available</string>
+    <string name="submission_status_card_body_fetching">The result is getting updated</string>
+    <string name="submission_status_card_body_unregistered">Help to break the chain of infection and warn others.</string>
+    <string name="submission_status_card_body_pending">The evaluation takes between 1-3 days.</string>
+    <string name="submission_status_card_body_invalid">Your test could not be evaluated.</string>
+    <string name="submission_status_card_body_positive">They tested positive for SARS-CoV-2.</string>
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/values/strings.xml:163]:
```suggestion
    <string name="submission_status_card_body_positive">You tested positive for SARS-CoV-2.</string>
```

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -113,77 +113,77 @@
...
+    <string name="submission_status_card_body_negative">They tested negative for SARS-CoV-2.</string>
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/values/strings.xml:164]:
```suggestion
    <string name="submission_status_card_body_negative">You tested negative for SARS-CoV-2.</string>
```

**Review by kolyaopahle — CHANGES_REQUESTED** [review]

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 6 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values/strings.xml]: Fixed this in the latest commit

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 7 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values/strings.xml]: Fixed this in the latest commit

**Comment by marcmuschko** [comment]
Dear @marcauberer ,
 
thank your very much for your contribution! The project team is working tirelessly on getting the app ready in time and working on many backlog items in parallel. This also means that some implementations cannot be done, yet, as we are, for example, waiting for final texts and translations from dedicated streams as these are not provided by the development team.
 
Therefore, we unfortunately have to reject your PR for now. 
 
To avoid this situation in the future, please open an issue for your fixes first, and wait until the team assigns you to the issue. We will further create issues with the community label, which you can claim by commenting on them and don't have to wait for the assignment.
 
Thank you for your understanding!

**Pull request closed without merging**